ext/drreg/drreg.c

```diff
@@ -78,6 +78,13 @@
 
 #define AFLAGS_SLOT 0 /* always */
 
+/* For simplicity, we assign aflags an alias that's one past the last gpr.
```

Comment:
IIRC, DR_REG_NULL is used to mean aflags in various places.  I would mention here that both aliases are used in different places to avoid confusion.

Reply:
Hmm... I think I should refactor the code that uses `DR_REG_NULL` for aflags to use `AFLAGS_ALIAS_REG` instead. I'll do it in a separate PR, as this one's already too big.